docs-src/astra-cli-core/modules/ROOT/pages/installation.adoc

```diff
@@ -280,13 +280,18 @@ Docker version 27.1.1, build 6312585
 ----
```

Comment:
For consistency, should this result snippet be introduced with `The command returns the Docker version that is installed:`?